src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostInitiateByIdControllerTest.java

```diff
@@ -674,7 +675,7 @@ void should_return_201_with_task_assigned_when_valid_old_assignee() throws Excep
     }
 
     @Test
-    void should_return_201_with_task_assigned_when_invalid_old_assignee_auto_assign_case() throws Exception {
+    void should_return_201_with_task_unassigned_when_invalid_old_assignee_auto_assign_case() throws Exception {
```

Comment:
This does not cover this AC -> If auto-assignable is true + authorisation is empty = assign task. Te test set state to UNASSIGNED + auto_assigned is false